src/ReportManager/EFBForm/fields/Section/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+import { FormSchemaContext } from '../../FormSchemaContext';
+import Header from '../Header';
+import Text from '../Text';
+
+const SECTION_ITEM_TYPES = {
+  HEADER: 'header',
+  FIELD: 'field'
+};
+
+const FORM_FIELDS = {
+  'TEXT': Text
+};
+
+const Section = ({ id }) => {
+  const { getSectionProps, getFieldProps, getHeaderProps } = useContext(FormSchemaContext);
+  const sectionProps = getSectionProps(id);
+
+  const renderColumnSectionItems = (column) => column.map(({ name, type }) => {
+    const isFieldType = type === SECTION_ITEM_TYPES.FIELD;
+    const itemProps = isFieldType
+      ? getFieldProps(name)
+      : getHeaderProps(name);
```

Comment:
Ok this answers one of my other questions, that's why the header was receiving its props already. But why not to keep the same pattern eveywhere? Just send the name to the component and let the component fetch its props? So we only have to: `<SectionItem fieldName={fieldName} key={name} />` and the SectionItem can read its own props.